src/System.Net.Security/src/System/Net/Windows/SSPIAuthType.cs

```diff
@@ -0,0 +1,213 @@
+                                        Interop.Secur32.Endianness.Native,
+                                        inputBuffer,
+                                        null,
+                                        outputBuffer,
+                                        ref outFlags
+                                        );
+        }
+
+        public int AcceptSecurityContext(SafeFreeCredentials credential, ref SafeDeleteContext context, SecurityBuffer[] inputBuffers, Interop.Secur32.ContextFlags inFlags, Interop.Secur32.Endianness endianness, SecurityBuffer outputBuffer, ref Interop.Secur32.ContextFlags outFlags)
+        {
+            return SafeDeleteContext.AcceptSecurityContext(ref credential, ref context, inFlags, endianness, null, inputBuffers, outputBuffer, ref outFlags);
+        }
+
+        public int InitializeSecurityContext(ref SafeFreeCredentials credential, ref SafeDeleteContext context, string targetName, Interop.Secur32.ContextFlags inFlags, Interop.Secur32.Endianness endianness, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, ref Interop.Secur32.ContextFlags outFlags)
+        {
+            return SafeDeleteContext.InitializeSecurityContext(ref credential, ref context, targetName, inFlags, endianness, inputBuffer, null, outputBuffer, ref outFlags);
+        }
+
+        public int InitializeSecurityContext(SafeFreeCredentials credential, ref SafeDeleteContext context, string targetName, Interop.Secur32.ContextFlags inFlags, Interop.Secur32.Endianness endianness, SecurityBuffer[] inputBuffers, SecurityBuffer outputBuffer, ref Interop.Secur32.ContextFlags outFlags)
+        {
+            return SafeDeleteContext.InitializeSecurityContext(ref credential, ref context, targetName, inFlags, endianness, null, inputBuffers, outputBuffer, ref outFlags);
+        }
+
+
+        public int EncryptMessage(SafeDeleteContext context, Interop.Secur32.SecurityBufferDescriptor inputOutput, uint sequenceNumber)
+        {
+            int status = (int)Interop.SecurityStatus.InvalidHandle;
+            bool ignore = false;
+
+            context.DangerousAddRef(ref ignore);
+            status = Interop.Secur32.EncryptMessage(ref context._handle, 0, inputOutput, sequenceNumber);
+            context.DangerousRelease();
+
+            return status;
+        }
+
+        public unsafe int DecryptMessage(SafeDeleteContext context, Interop.Secur32.SecurityBufferDescriptor inputOutput, uint sequenceNumber)
+        {
+            int status = (int)Interop.SecurityStatus.InvalidHandle;
+            uint qop = 0;
+
+            try
+            {
+                bool ignore = false;
+                context.DangerousAddRef(ref ignore);
+                status = Interop.Secur32.DecryptMessage(ref context._handle, inputOutput, sequenceNumber, &qop);
+            }
+            finally
+            {
+                context.DangerousRelease();
+            }
+
+            const uint SECQOP_WRAP_NO_ENCRYPT = 0x80000001;
```

Comment:
Original code in master:          
`            if (status == 0 && qop == Interop.Secur32.SECQOP_WRAP_NO_ENCRYPT)`

I don't understand why this is changed. Maybe this was an older version, prior to my commit to master?